saleor/order/models.py

```diff
@@ -108,7 +108,7 @@ class Order(ModelWithMetadata, ModelWithExternalReference):
     id = models.UUIDField(primary_key=True, editable=False, unique=True, default=uuid4)
     number = models.IntegerField(unique=True, default=get_order_number, editable=False)
     use_old_id = models.BooleanField(default=False)
-    created_at = models.DateTimeField(default=now, editable=False)
+    created_at = models.DateTimeField(default=now, editable=False, db_index=True)
```

Comment:
Please use `Meta.indexes` instead of  `db_index`. 

And in migration you need to use `concurrently` build, you can refer to: https://docs.saleor.io/docs/3.x/developer/community/zero-downtime-migrations#adding-index-to-the-database

Reply:
Hi @kadewu , I have made these changes. Also could you point me to some docs for list of pre-checks/tests we could run in local.

Reply:
Hi @ritanjandawn in our repository we are using [`pre-commit`](https://docs.saleor.io/docs/3.x/developer/community/contributing#coding-stylel) and [`pytest`](https://docs.saleor.io/docs/3.x/developer/community/contributing#how-to-run-tests).